coursework/mobile/index.md

```diff
@@ -1406,12 +1435,14 @@ contains the following:
     your answers to all compulsory milestones that require
     written answers.
 
-3.  A one/two-page pitch of your application,
+3.  Lighthouse `html` report, `group-<number>-lighthouse.html`.
```

Comment:
While you're at this, can help change all the bullet points to `1.`? They'll increment automatically and if we insert in the middle we don't have to update the rest.